packages/@aws-cdk/aws-iam/test/policy-document.test.ts

```diff
@@ -572,4 +591,197 @@ describe('IAM polocy document', () => {
 
     expect(stack.resolve(doc1)).toEqual(stack.resolve(doc2));
   });
+
+  describe('from JSON', () => {
+    test('no principal', () => {
+      const stack = new Stack();
```

Comment:
We like the "GIVEN/WHEN/THEN" pattern for tests:

- GIVEN: prepare the conditions for the test
- WHEN: the actual things that's being tested
- THEN: the assertions.

See examples across the code base for how this is formatted